content/en/docs/reference/glossary/kubelet.md

```diff
@@ -14,4 +14,4 @@ tags:
 
 <!--more-->
 
-The kubelet takes a set of PodSpecs that are provided through various mechanisms and ensures that the containers described in those PodSpecs are running and healthy. The kubelet doesn't manage containers which were not created by Kubernetes.
+The [kubelet](/docs/reference/command-line-tools-reference/kubelet/) takes a set of PodSpecs that are provided through various mechanisms and ensures that the containers described in those PodSpecs are running and healthy. The kubelet doesn't manage containers which were not created by Kubernetes.
```

Comment:
Please wrap long lines like this since you are working on it?

Reply:
@tengqm 
wrapped long lines